parser.js

```diff
@@ -274,7 +274,7 @@
     function createClassRange(min, max, from, to) {
       // See 15.10.2.15:
       if (min.codePoint > max.codePoint) {
-        throw SyntaxError('invalid range in character class');
+        bail('invalid range in character class: ' + min.raw + '-' + max.raw, from, to);
```

Comment:
Great. Also happy to chim in and do a bit of work here if you want - just let me know :)

Have fun!

On August 11, 2015 11:18:19 PM GMT+02:00, Kevin Decker <notifications@github.com> wrote:
>> @@ -274,7 +274,7 @@
>>      function createClassRange(min, max, from, to) {
>>        // See 15.10.2.15:
>>        if (min.codePoint > max.codePoint) {
>> -        throw SyntaxError('invalid range in character class');
>> +        bail('invalid range in character class: ' + min.raw + '-' +
>max.raw, from, to);
>
>Ok, I'll give this a spin and ping here when done.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/jviereck/regjsparser/pull/74/files#r36800317

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.